colabcode/code.py

```diff
@@ -48,7 +48,8 @@ def __init__(
     def _install_code():
         subprocess.run(["wget", "https://code-server.dev/install.sh"], stdout=subprocess.PIPE)
         subprocess.run(
-            ["sh", "install.sh", "--version", f"{CODESERVER_VERSION}"],
+            # ["sh", "install.sh", "--version", f"{CODESERVER_VERSION}"],
```

Comment:
Overall Code can be improved.
output of subprocess.run command is not taken care of.
Comments can be removed.

def _install_code():
    result = subprocess.run(["wget", "https://code-server.dev/install.sh"], stdout=subprocess.PIPE)
    if result.returncode != 0:
        raise RuntimeError("Failed to download installation script")
        
    result = subprocess.run(["sh", "install.sh"], stdout=subprocess.PIPE)
    if result.returncode != 0:
        raise RuntimeError("Failed to run installation script")